src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -266,3 +267,14 @@ export function cleanEnvironment<T>(spec: T): T {
 
     return copy as T;
 }
+
+export function isLocalLaunch(configuration: IConfigurationService) {
```

Comment:
Ideally this should be a value should be a constant, something that needs to be injected into our DI container like the following
```typescript
serviceManager.addSingletonInstance<boolean>(IsLocalLaunch, true or false);
```

As this cannot change in the same session of VS Code.& if the user changes it they must reload VS Code to get things working.
This way, we don't need to keep checking values all over the place for `isLocalLaunch` etc..
Its not a mandatory change & its a cleanup that we can do later.